aodndata/acorn/handler.py

```diff
@@ -109,7 +109,12 @@ def preprocess(self):
             creation_date_nc_s3 = get_creation_date(os.path.join(self.opendap_root, destination_s3))
 
             if creation_date_nc_pipeline < creation_date_nc_s3:
+                self.logger.warning("incoming file '{dest_path}': creation date is older than file already on storage".
+                                 format(dest_path=nc_file.name))
                 nc_file.publish_type = PipelineFilePublishType.NO_ACTION
+            else:
+                self.logger.info("incoming file '{dest_path}': creation date is newer/equal to file already on storage".
+                                 format(dest_path=nc_file.name))
```

Comment:
This is the expected use case, I don't think it's worth logging this, this would make the logs more noisy than anything.